ztag/transform.py

```diff
@@ -63,7 +63,10 @@ def transform(self, obj):
         try:
             return self._transform_object(obj)
         except (KeyError, TypeError, IndexError) as e:
-            raise IgnoreObject(original_exception=e)
+            import traceback
+            exc_info = traceback.format_exc()
+            raise IgnoreObject(e, exc_info)
+
```

Comment:
This is certainly right in the realm of zgrab2, where each scan has its own schema under its own top-level key -- so `TypeError`s, `KeyErrors` etc do indicate something seriously wrong. If you explicitly have a case where you want a response to be ignored after it's already gotten to your transform, you need to explicitly raise an `IgnoreObject`.

But, I'm a bit concerned about legacy zgrab transforms; it seems like there were some transforms that were simplified via an unchecked `KeyError` leading to a short-circuit / ignore.

Having said that, if logs don't blow up after this, this would put us on the right track going forward.